animation_nodes/nodes/matrix/distribute_matrices.pyx

```diff
@@ -112,6 +116,9 @@ class DistributeMatricesNode(bpy.types.Node, AnimationNode):
             self.newInput("Float", "End Size", "endSize", value = 0.5, minValue = 0)
             self.newInput("Float", "Start Angle", "startAngle", value = 0)
             self.newInput("Float", "End Angle", "endAngel", value = 6 * PI)
+            self.newInput("Float", "Altitude", "altitude", value = 0)
+            self.newInput("Interpolation", "Interpolation_1", "interpolation_1", defaultDrawType = "PROPERTY_ONLY", hide = True)
+            self.newInput("Interpolation", "Interpolation_2", "interpolation_2", defaultDrawType = "PROPERTY_ONLY", hide = True)
```

Comment:
You should give better names to those.

Reply:
is like this `interpolation_XY` and `interpolation_Z` ?

Reply:
`radiusInterpolation` and `zInterpolation`.

Reply:
Hi, I agree with Omar's suggestions, but since we are naming "Height" as "Height" not "z", perhaps it makes more sense and consistent to name it "heightInterpolation" instead of "zInterpolation". This is a really minor issue though, so I don't mind which name to use eventually, because I understand "heightInterpolation" maybe a little bit too long?

Reply:
Sure, doesn't matter that much.